.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
hmmm... I actually completely disabled dependabot because it complains for too many things that are not important for the gem itself. Just too annoying to keep up with it 🙃

Reply:
Just noticed (thanks @benkoshy) that is checking the github-actions, which is actually very useful!